src/components/QRCodeSurface/index.js

```diff
@@ -14,7 +14,9 @@ const defaultProps = {
 };
 
 const QRCodeSurface = ({ children, size, title, xmlns, ...props }) => (
-  <svg {...props} height={size} width={size} xmlns={xmlns}>
+  const viewBox = "0 0 " + size + " " + size;
+  
+  <svg {...props} height={size} width={size} viewBox={viewBox} xmlns={xmlns}>
```

Comment:
I'm not sure this is needed. There is still `...props` here, so passing `viewBox` should work even without this change, right?

Reply:
Here is a commit with source code and updated demo: https://github.com/rosskhanas/react-qr-code/commit/1f43adb3b41305b9a3ab004b337b3bd34bf9fe6c

Reply:
I'm fine if that is the goto way for it. Feel free then to close the PR.